certified-connectors/Tribal/README.md

```diff
@@ -0,0 +1,70 @@
+# Tribal Connector
+
+Tribal Edge modules, such as Tribal Admissions, provide staff and students the tools they need to succeed in the world of education. Connect to Tribal Edge modules using the Tribal Connector to manage information, create clearance checks for applications, and much more.
+
+## Tribal
```

Comment:
@mszy-tribal Please add the section name mentioned in the template, like below:

`"Publisher: Tribal"`

Reply:
@sriyen-msft 

The section is there, it is below, it just doesn't have the word Publisher in it and neither do any of the other readmes in the repository.

Reply:
I have added the word publisher